joomla/joomla-cms
# TinyMCE xtd-button close (again, really) and drop jquery

Pull Request for Issue # .
### Summary of Changes
- Drag and drop required jQuery (which wasn't really loaded by the plugin) which was WRONG. Blame me for that!
- New tinymce-init relied on jQuery (again no jQuery load call)
- jModalClose relied on `document.onreadystatechange` which was broken by the core.js with the new Joomla.Options.
### Testing Instructions

Go to Admin -> configuration -> articles ->editing layouts and disable (hide) Frontend Images and Links.
Try to edit an article in the front end and use any of the xtd-buttons that have a modal (e.g. menu)
The modal will not close when selecting a menu.

Apply this patch and retry

Also try to drag and drop an image to tinymce area, the progress bar should appear and when complete disappear.
### Documentation Changes Required

None

@Fedik you had a line `options = jQuery.extend({}, options);` in tiny-init.js. Is that required? Do we have to replicate this with native JS?
Also I moved these to files to system/js and this will make sense for J4 as all the Joomla owned scripts will be in that directory and everything else in a media/vendor/...

@wilsonge @rdeutz this is a release-blocker IMHO
